src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -4638,6 +4648,56 @@ def health(self) -> Dict[str, Any]:
+            # Let any other owner (provider/model auth, SecretRef resolution,
+            # MCP capability load) that recorded into a shared registry surface
+            # here too, if one was attached to this gateway instance.
+            shared = getattr(self, "_degraded_registry", None)
+            if shared is not None:
+                for owner in shared.list_degraded():
+                    registry.mark(owner)
+            degraded_owners = registry.to_list()
+            if degraded_owners:
+                result["degraded_owners"] = degraded_owners
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)